modules/cluster-logging-collector-log-forward-fluentd.adoc

```diff
@@ -81,3 +81,32 @@ spec:
 ----
 $ oc create -f <file-name>.yaml
 ----
+
+[id="cluster-logging-collector-log-forward-fluentd-config]
+== Enabling nanosecond_precision for Logstash to ingest data from fluentd
+For Logstash to ingest log data from fluentd, you must enable nanosecond precision in the Logstash configuration file.
+
+.Procedure
+* In the Logstash configuration file,  set `{ nanosecond_precision => true }`.
+
+.Example Logstash configuration file
+[source,yaml]
+----
+  input {
```

Comment:
I'm not familiar with logstash config, but this looks odd  - the indenting and line breaks are a bit random, it's not well formed YAML, it looks a bit like Ruby Hash syntax but it's not well formed for that either. Is this example correct?

Reply:
@anpingli can you provide a sample logstash config file please? this one seems wrong.

Reply:
@libander  The example is correct.  The logstash configuration file is not yam.

Reply:
To be more clear, we can use content as below:

```
input { tcp { codec => fluent { nanosecond_precision => true } port => 24114 } }
filter { }
output { stdout { codec => rubydebug } }
```

Reply:
+1 - I was confused by the `[source,yaml]` tag on line 93 - that should be changed to indicate the actual format,  or to just `[source]` if the doc generator doesn't know about the logstash format - it is not like any common config format that I recognize.